playbooks/roles/os_temps/tasks/main.yml

```diff
@@ -41,11 +41,17 @@
 - name: "Create project {{ openshift_project }}"
   import_tasks: get_set_project.yml
 
-# Import_tasks jq for querying container config files
+# Check if jq is already installed so we can skip installation
+- name: "Check if jq is already installed"
+  command: jq --version
+  register: jq_installed
+  ignore_errors: yes
+
+# Install jq for querying container config files
 - name: "Install jq for querying container config files"
   package:
     name: jq
-  when: host_os == "linux"
+  when: host_os == "linux" and jq_installed.stdout is not defined
```

Comment:
same, I think jq_installed == 0 could be better

Reply:
I agree, might be better to look at rc here